pennylane/gradients/parameter_shift.py

```diff
@@ -368,24 +378,31 @@ def var_param_shift(tape, argnum, shifts=None, gradient_recipes=None, f0=None):
         gradient_tapes.extend(pdA2_tapes)
 
     def processing_fn(results):
+        # Apply the same squeezing as in qml.QNode to make the transform output consistent.

```

Comment:
```suggestion
        # HOTFIX: Apply the same squeezing as in qml.QNode to make the transform output consistent.
```

Reply:
This then also would apply in the comment in `expval_param_shift` and `compute_hessian_tapes`, right?